.travis.yml

```diff
@@ -10,18 +10,19 @@ env:
 
 matrix:
   include:
-    - os: linux
-      python: "3.6"
+    - python: 3.6
+      os: linux
       addons:
         apt:
           sources:
             - ubuntu-toolchain-r-test     # For gcc 4.9, 5 and 7
           packages:
-            - gcc-4.9
-            - g++-4.9
-      env: DEVITO_ARCH=gcc-4.9 DEVITO_OPENMP=0 INSTALL_TYPE=pip_setup RUN_EXAMPLES=False MPI_INSTALL=1
-    - os: linux
-      python: "3.6"
+            - gcc-7
```

Comment:
why do you wanna drop 4.9? it's still (unfortunately) one of the most used gcc versions, so it should be tested right?

what we may want is testing gcc9 ...

Reply:
4.9 - Lack of a build for bionic, there only appears to be a stub set of packages on the system set, and none on ubuntu-toolchain-r, but I haven't looked ultra-carefully.

9 - I think this should work with some bash conditionals wrapped round the gcc install to only turn on the ubuntu-toolchain-r repo when installing 9.

I'm a bit tied up with things this week but will do my best to fit in having a look at it if nobody beats me to it.